state-loader.md

```diff
@@ -251,50 +227,46 @@ The `"rlp"` key loads the block information.
           ...
           </k>
 
-    rule <k> load "transaction" : { ACCTID: { KEY : VALUE , REST } }
-          => load "transaction" : { ACCTID : { KEY : VALUE } }
-          ~> load "transaction" : { ACCTID : { REST } }
-          ...
-         </k>
-      requires REST =/=K .JSONList
-
-    rule <k> load "transaction" : { TXID : { GLIMIT     : TG:Int } } => . ... </k>
+    syntax EthereumCommand ::= "loadTransaction" Int JSON
+ // -----------------------------------------------------
+    rule <k> loadTransaction TXID { GLIMIT     : TG:Int, REST } => loadTransaction TXID { REST } ... </k>
          <message> <msgID> TXID </msgID> <txGasLimit> _ => TG </txGasLimit> ... </message>
       requires GLIMIT in (SetItem("gas") SetItem("gasLimit"))
 
-    rule <k> load "transaction" : { TXID : { "gasPrice" : TP:Int } } => . ... </k>
+    rule <k> loadTransaction TXID { "gasPrice" : TP:Int, REST } => loadTransaction TXID { REST } ... </k>
          <message> <msgID> TXID </msgID> <txGasPrice> _ => TP </txGasPrice> ... </message>
 
-    rule <k> load "transaction" : { TXID : { "nonce" : TN:Int } } => . ... </k>
+    rule <k> loadTransaction TXID { "nonce" : TN:Int, REST } => loadTransaction TXID { REST } ... </k>
          <message> <msgID> TXID </msgID> <txNonce> _ => TN </txNonce> ... </message>
 
-    rule <k> load "transaction" : { TXID : { "value" : TV:Int } } => . ... </k>
+    rule <k> loadTransaction TXID { "value" : TV:Int, REST } => loadTransaction TXID { REST } ... </k>
          <message> <msgID> TXID </msgID> <value> _ => TV </value> ... </message>
 
-    rule <k> load "transaction" : { TXID : { "to" : TT:Account } } => . ... </k>
+    rule <k> loadTransaction TXID { "to" : TT:Account, REST } => loadTransaction TXID { REST } ... </k>
          <message> <msgID> TXID </msgID> <to> _ => TT </to> ... </message>
 
-    rule <k> load "transaction" : { TXID : { "data" : TI:ByteArray } } => . ... </k>
+    rule <k> loadTransaction TXID { "data" : TI:ByteArray, REST } => loadTransaction TXID { REST } ... </k>
          <message> <msgID> TXID </msgID> <data> _ => TI </data> ... </message>
 
-    rule <k> load "transaction" : { TXID : { "v" : TW:Int } } => . ... </k>
+    rule <k> loadTransaction TXID { "v" : TW:Int, REST } => loadTransaction TXID { REST } ... </k>
          <message> <msgID> TXID </msgID> <sigV> _ => TW </sigV> ... </message>
 
-    rule <k> load "transaction" : { TXID : { "r" : TR:ByteArray } } => . ... </k>
+    rule <k> loadTransaction TXID { "r" : TR:ByteArray, REST } => loadTransaction TXID { REST } ... </k>
          <message> <msgID> TXID </msgID> <sigR> _ => TR </sigR> ... </message>
 
-    rule <k> load "transaction" : { TXID : { "s" : TS:ByteArray } } => . ... </k>
+    rule <k> loadTransaction TXID { "s" : TS:ByteArray, REST } => loadTransaction TXID { REST } ... </k>
          <message> <msgID> TXID </msgID> <sigS> _ => TS </sigS> ... </message>
+
+    rule <k> loadTransaction _ { .JSONList } => . ... </k>
```

Comment:
I like to put the base case first, because it's the simplest and will be overlooked if it's at the end. I also usually put `[owise]` rules first since they're unique (or more unique than the normal rules), and somehow also a base-case.

Maybe we should also have a rule:

```
    rule <k> loadTransaction _ { _:String : _ , REST => REST } ... </k> [owise]
```

Reply:
The presence of an `[owise]` rule like that seems to be what breaks the conformance test on the ocaml backend, so I left it out.